### DISCUSSION TITLE
Punctuation Restoration: which languages are supported?

### DISCUSSION THREAD

**Discussion by dgoryeo** [PaddlePaddle/PaddleSpeech#3022] [Q&A]
Does the released mode of [Punctuation Restoration](https://github.com/PaddlePaddle/PaddleSpeech#PunctuationRestoration) support Japanese? 
Thanks

**Comment by lizezheng** [comment]
Not yet, stay tuned please

**Reply by dgoryeo** [reply]
Hi, going back to this old thread. By any chance, is Japanese language now supported for Punctuation Restoration? Thanks.
